src/Cardano/Wallet/BlockSyncer.hs

```diff
@@ -0,0 +1,64 @@
+{-# LANGUAGE ScopedTypeVariables #-}
+
+-- |
+-- Copyright: © 2018-2019 IOHK
+-- License: MIT
+--
+-- This module contains the ticking function that is responsible for invoking block acquisition functionality and executing it in periodic fashion.
+--
+-- Known limitations: ticking function makes sure action is not executed on already consumed block, but does not check and handle block gaps (aka catching up)
+
+module Cardano.Wallet.BlockSyncer
+  (
+    BlockHeadersConsumed(..)
+  , tickingFunction
+  ) where
+
+
+import Prelude
+
+import Cardano.Wallet.Primitive
+    ( Block (..), BlockHeader )
+import Control.Concurrent
+    ( threadDelay )
+import Data.Time.Units
+    ( Second, toMicroseconds )
+
+import qualified Data.List as L
+
+
+newtype BlockHeadersConsumed = BlockHeadersConsumed [BlockHeader] deriving (Show, Eq)
```

Comment:
https://github.com/input-output-hk/cardano-wallet/wiki/Coding-Standards#limit-line-length-to-80-characters

Propose:
```
newtype BlockHeadersConsumed =
    BlockHeadersConsumed [BlockHeader]
    deriving (Show, Eq)
```

Reply:
:+1: